engine/src/main/java/com/redhat/cloud/notifications/recipients/itservice/ITUserService.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.cloud.notifications.recipients.itservice;
+
+import com.redhat.cloud.notifications.recipients.itservice.pojo.request.ITUserRequest;
+import com.redhat.cloud.notifications.recipients.itservice.pojo.response.ITUserResponse;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@RegisterRestClient(configKey = "it-s2s")
+public interface ITUserService {
+
+    @POST
+    @Path("/findUsers")
+    @Consumes(MediaType.APPLICATION_JSON)
+    List<ITUserResponse> getUsers(ITUserRequest itUserRequest);
```

Comment:
`@Produces(MediaType.APPLICATION_JSON)` is missing here.

Reply:
added in #1045